install/framework/nunit-framework.wxs

```diff
@@ -58,7 +58,7 @@
     
 	<Fragment>
 		<Directory Id="TARGETDIR" Name="SourceDir">
-			<Directory Id="ProgramFilesFolder">
+			<Directory Id="ProgramFiles64Folder">
```

Comment:
Moved the install to `Program Files` instead of `Program Files (x86)`. I see no reason it should not be here?